Doc/library/annotationlib.rst

```diff
@@ -368,6 +368,9 @@ Functions
      doesn't have its own annotations dict, returns an empty dict.
    * All accesses to object members and dict values are done
      using ``getattr()`` and ``dict.get()`` for safety.
+   * For :class:`functools.partial` and :class:`functools.partialmethod` objects,
+     only returns annotations for parameters that have not been bound by the
+     partial application, along with the return annotation if present.
```

Comment:
With the new implementation, I’m not sure how much `annotationlib` docs should mention support for `partial`. Maybe mention the generic support and link to a new section in `functools` doc with the info here?

Reply:
Yes! I have updated the docs